enterprise-suite/es-monitor-api/bootstrap-monitors.json

```diff
@@ -51,7 +51,7 @@
       "model": "threshold",
       "parameters": {
         "name": "node_network_errors",
-        "metric": "node_network_errors",
+        "metric": "node_network_errors_rate",
```

Comment:
There is no disagreement.. I debated this over..looked around and kept it inline with other names.

container_restarts_rate
kube_pod_container_restarts_rate